src/catalogue/category/catalogueCategoryDialog.component.tsx

```diff
@@ -159,6 +165,29 @@ const CatalogueCategoryDialog = React.memo(
       if (errorIndexes.length !== 0) {
         hasErrors = true;
       }
+
+      if (categoryData.catalogue_item_properties) {
+        const listOfPropertyNames: string[] =
+          categoryData.catalogue_item_properties.map((property) =>
+            property.name.toLowerCase()
+          );
+        const uniqueNames = new Set();
+        const duplicateNames: string[] = [];
+
+        for (const name of listOfPropertyNames) {
+          if (uniqueNames.has(name)) {
+            duplicateNames.push(name);
+          } else {
+            uniqueNames.add(name);
+          }
+        }
+        if (duplicateNames.length > 0) {
+          setDuplicatePropertyError(duplicateNames);
+          hasErrors = true;
+        }
```

Comment:
![dup item prop name](https://github.com/ral-facilities/inventory-management-system/assets/83226114/d68bbdda-a1e0-4ba0-a654-13c5a27bd7de)

The property names needs to be trimmed before being checked. Currently if you have "name" and "name " (with a space at the end) they are two different values. same as the back-end change i suggested